src/init/features/hosting/github.ts

```diff
@@ -184,9 +186,27 @@ export async function initGitHub(setup: Setup, config: any, options: any): Promi
   logLabeledBullet("Action required", `Push any new workflow file(s) to your repo`);
 }
 
+function getGitFolderPath(): string {
+  const commandDir = process.cwd();
+  let projectRootDir = commandDir;
+
+  while (!fs.existsSync(path.resolve(projectRootDir, ".git"))) {
+    const parentDir = path.dirname(projectRootDir);
+    if (parentDir === projectRootDir) {
```

Comment:
And stop case when it can't go up higher? This is a little unclear, might want to add a comment.

Reply:
FYI testing Windows paths show that stopcase is `.`, e.g, `path.dirname('C://')` => `"."`, this logic might still work but I don't know if what `.` will equate too in powershell. You might get undefined behavior.

Reply:
Added a comment. Since this is really similar to the code that finds `firebase.json`, I think we're ok here